includes/definitions/discovery/vrp.yaml

```diff
@@ -1,7 +1,7 @@
 mib: HUAWEI-WLAN-CONFIGURATION-MIB:HUAWEI-WAN-MIB:HUAWEI-ENTITY-EXTENT-MIB:HUAWEI-ENERGYMNGT-MIB:HUAWEI-STACK-MIB
 modules:
     os:
-        sysDescr_regex: '/\((?<hardware>\S+) (?<version>V[0-9]{3}R[0-9]{3}[0-9A-Z]+)/'
+        sysDescr_regex: '/\((?<hardware>[^ V]+( (?!V)[^ V]*)*) (?<version>V[0-9]{3}R[0-9]{3}[0-9A-Z]+)/'
```

Comment:
Maybe a simpler fix is something like `\((?<hardware>[^V\)]+)(?<version>V[^)]+)\)`
Havent checked if all devices follow the pattern of `(*hardware* V*version*)` tho

Reply:
how about .*

Reply:
`\((?<hardware>[^)]+) (?<version>V[0-9]{3}R[0-9]{3}[0-9A-Z]+)`

Reply:
A little more generic version of Jellyfrog's, `\((?<hardware>[^\)]+)(?<version>V[^)]+)\)`  146 steps

Reply:
Right, both of you ! Never, ever, play with regexp at "farTooLate oclock"... running the checks locally succeed with test data. Should be all right with the PR.